docs/JSON-RPC.md

```diff
@@ -29,6 +29,8 @@ The JSON-RPC server is based on the [JSON-RPC 2.0](https://www.jsonrpc.org/speci
 - A **response** from Jamulus to the consumer.
 - A **notification** from Jamulus to the consumer.
 
+**Tip:** On Linux, you can connect to a JSON-RPC server using the `nc` CLI tool. On Windows, you can download and use the `ncat` CLI tool.  Also [see this page](https://github.com/jamulussoftware/jamulus/pull/1975) for a connection example using Node.js.
```

Comment:
I think something clearer than a reference to a pull request is needed.  Extracting the example into a [gist](https://gist.github.com/) might be clearer.

Reply:
> Extracting the example into a [gist](https://gist.github.com/) might be clearer.

Have we published documentation snippets to gist before? I can [make one in my account](https://gist.github.com/mcfnord/bea59c2d71c789b420821fb0629e9c4e), but that's not making one in this repo.

Reply:
What speaks against including it into this document?

Reply:
> What speaks against including it into this document?

I wasn't sure (as I couldn't find it) how big it was.